static/app/stores/groupStore.tsx

```diff
@@ -92,13 +74,15 @@ interface GroupStoreDefinition extends StoreDefinition, InternalDefinition {
     itemIds: string[],
     response: Partial<Group>
   ) => void;
+
```

Comment:
Did you mean to add these empty lines here?

Reply:
yeah, it was hard to read before I think